tools/2020/KeymanUrls.php

```diff
@@ -0,0 +1,35 @@
+<?php
+  declare(strict_types=1);
+
+  namespace Keyman\Site\com\keyman\api;
```

Comment:
Is there a standard on namespace matching the folder paths? something along the lines of `\Keyman\tools\2020`?

Reply:
The folder path is a little inconsistent. Basically, the `/tools/2020` folder is the current root folder for `Keyman\Site\com\keyman\api` namespace for api.keyman.com. This path may change in future as we remove legacy code or translate it into the namespaced model. On other sites, we have slightly different folder layouts, for historical reasons.

We also have `/_common` for `Keyman\Site\Common`.

These are controlled in `/tools/autoload.php` so there is only one place we need to change them in the future.